src/theory/quantifiers/proof_checker.cpp

```diff
@@ -97,6 +97,7 @@ Node QuantifiersProofRuleChecker::checkInternal(
     else
     {
       std::vector<Node> echildren(children[0][0].begin(), children[0][0].end());
+      echildren[1] = echildren[1].notNode();
```

Comment:
what is this for?

Reply:
The rule was incorrect before, it was converting `~forall x. P` to `exists x. P` when it should have been `exists x. ~P`.

Reply:
Ah, I see.